### ISSUE TITLE
Example for integration

### ISSUE THREAD

**Issue by Testato** [add-ons/service.iptv.manager#64]
I want enable two of mine kodi addon for IPTV Manager.
I readed your developer section but it is not simple understand how implement it.

May we have a complete Basic example of a working iptv manager addon ?
I mean a real basic .zip addon that we can simply install on kodi, and use it for test, and change for learn the implementation.

Thanks

**Comment by michaelarnauts** [comment]
I think one of the simplest add-ons that currently is using IPTV Manager is https://github.com/add-ons/plugin.video.regiotv/, but that doesn't do EPG data. However, I think if you've managed to make it work for just the channels, the EPG itself isn't that hard.

You should try to follow the steps on https://github.com/add-ons/service.iptv.manager/wiki/Integration, feel free to ask questions here or improve the wiki in case things aren't clear.

**Comment by dagwieers** [comment]
@michaelarnauts I think [plugin.audio.vrt.radio](https://github.com/add-ons/plugin.audio.vrt.radio) may be a better example, given the add-on is quite straightforward and it does include EPG data.

**Comment by dagwieers** [comment]
Thinking about this, I prefer we use the Red Bull TV add-on as the example implementation. Since it is an internationally known add-on, the content is open to anyone and high quality even, and people can relate to the implementation. (REST API)

We may want to document this add-on a bit better or review it to make it self-explanatory.
cc @piejanssens

**Comment by dagwieers** [comment]
@Testato If you send me your email address privately, I can add you to our Add-ons Slack channel and we can collaborate in real time.

**Comment by Testato** [comment]
Thanks,
Where i can found your email ?

**Comment by dagwieers** [comment]
On my GitHub profile, click my name :point_up:

**Comment by dagwieers** [comment]
I added 2 new sections to the Integration documentation:
- [Reference implementation](https://github.com/add-ons/service.iptv.manager/wiki/Integration#reference-implementation)
- [Contact](https://github.com/add-ons/service.iptv.manager/wiki/Integration#contact)

The reference implementation docs have links to each of the files we have modified. The links jump to the code sections, however that is a bit brittle as it is by line number. I think we can close this ticket now.

@Testato If you still have questions, feel free to let us know, or open a ticket for it.
